Adyen/Core/Payment Methods/PreselectedPaymentMethod.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// A payment method thats to be shown as preselected payment method that is ready to be submitted.
```

Comment:
Not sure I fully understand this description 🤔

Reply:
I renamed it and changed the docs, let me know if its still vague

Reply:
Maybe ` A payment method that is ready for instant submission. Could be displayed as preselected payment method`